internal/cli/atlas/privateendpoints/datalake/aws/list.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aws
+
+import (
+	"context"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+var listTemplate = `ID	ENDPOINT PROVIDER	TYPE	COMMENT{{range .Results}}
+{{.EndpointID}}	{{.Provider}}	{{.Type}}	{{.Comment}}{{end}}
+`
+
+type ListOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store store.DataLakePrivateEndpointLister
+}
+
+func (opts *ListOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+func (opts *ListOpts) Run() error {
+	r, err := opts.store.DataLakePrivateEndpoints(opts.ConfigProjectID())
+
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas privateEndpoint(s)|privateendpoint(s) dataLakes aws list|ls [--projectId projectId].
+func ListBuilder() *cobra.Command {
+	opts := new(ListOpts)
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List Data Lake Private Endpoints for your project.",
```

Comment:
Just a suggestion to add a little more info:
```suggestion
		Short:   "List Data Lake private endpoints for your project.",
		Long:    "Your API key must have the GROUP_READ_ONLY (Project Data Access Read Only) or higher role to list private endpoints."
```

Reply:
I don't think this change made it in (the additional Long description).